labs/lab-03/tasks/goto-algs/binary_search/tests/test_binary_search.c

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+
+#include "graded_test.h"
+
+int binary_search(int *v, int len, int dest);
+
+static int test_1(void)
```

Comment:
```suggestion
static int test_15_found(void)
```
This makes it easier for students to figure out what the test is about just by running it. And make sure to update the `README.md` file with these changes.